HPXMLtoOpenStudio/resources/hpxml.rb

```diff
@@ -9702,16 +9702,23 @@ def initialize(hpxml_element, *args, **kwargs)
     # @return [Array<HPXML::ElectricPanel>] The list of attached electric panels
     def sub_panels
       list = []
-      @parent_object.electric_panels.each do |electric_panel|
-        next unless @id == electric_panel.id
```

Comment:
Am I right that we don't need this piece? We already get _this_ electric panel's branch circuits by calling just `branch_circuits.xxx`?